product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/index.mdx

```diff
@@ -4,9 +4,52 @@ redirects:
+
+-   You need to configure the PrivateLink only once. Then you can use multiple VPC endpoints to connect applications from different VPCs.
+-   There's no risk of IP address conflicts.
+
+There's an associated cost of resources, however.
+
+For more information, see [VPC endpoint services (AWS PrivateLink)](https://docs.aws.amazon.com/vpc/latest/privatelink/endpoint-service.html).
+
+The way you create a private endpoint differs when you are using your own AWS account as opposed to using BigAnimal's cloud account.
+
+## Using BigAnimal's cloud account
+
+When using BigAnimal's cloud account, you provide BigAnimal with your AWS account ID when creating a cluster. BigAnimal, in turn, provides you with a AWS service name, which you can use to connect to your cluster privately.
+
+1. When creating your cluster, on the **Cluster Settings** tab, in the **Network Connectivity** section:
+   1. Select **Private**.
+
+   1. Enter your application's AWS account ID.
+
+1. After the cluster is created, go to its cluster details to see the corresponding endpoint service name. You need the service name while creating a VPC endpoint.
+
+1. Create a VPC endpoint in the client's VPC. The steps for creating a VPC endpoint in the client's VPC are the same  whether you are using BigAnimal's cloud or your own. The steps are available [here](#step-2-create-a-vpc-endpoint-in-the-clients-vpc).
+
+1. In your application's AWS account, select **VPC** and then select **Endpoints**. Select the endpoint you created previously and use the service name provided in the details section in BigAnimal to access your cluster.
+
+   ```shell     
+   # Replace the port 5432 below if that has been changed in the Big Animal DB Configuration
+   psql -h vpce-XXXXXXXXXXXXXXXXXXXX.eu-west-1.vpce.amazonaws.com -U edb_admin -p 5432     
+    __OUTPUT__
+    Password for user edb_admin: 
+    ```
+    ``` 
+    psql (15.2)
+    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off)
+    Type "help" for help.
+
+    edb_admin=>
+    ``` 
+## Using your own AWS account
+
 Two different methods enable you to connect to your private cluster from your application's VPC in AWS. Each method offers different levels of accessibility and security. The VPC endpoint method is recommended and is most commonly used. However, you can also use the VPC peering connection method if required by your organization.
 
-## AWS VPC endpoint (recommended)
+### AWS VPC endpoint (recommended)
 
 AWS VPC endpoint (AWS Private Link) service is a network interface that securely connects a private IP address from your AWS VPC to an external service. You grant access only to a single cluster instead of the entire BigAnimal resource VPC, thus ensuring maximum network isolation. Other advantages include:
 
```

Comment:
@valeriodelsarto @drothery-edb The advantages below are stated at the start of this section.  Does it need to be repeated?  Or should the opening paragraph discuss the advantages/disadvantages of VPC endpoints vs VPC peering?  As of now, the opening paragraph focuses on vpc endpoints, the preferred approach.

Reply:
removed redundant info